internal/tfimport/tfimport.go

```diff
@@ -186,46 +347,57 @@ var importers = map[string]resourceImporter{
 		Tmpl:   "projects/{{.project}}/instances/{{.name}}",
 	},
 	"google_sql_user": &importer.SQLUser{},
-	"google_iap_tunnel_instance_iam_policy": &importer.SimpleImporter{
-		Fields: []string{"project", "zone", "instance"},
-		Tmpl:   "projects/{{.project}}/iap_tunnel/zones/{{.zone}}/instances/{{.instance}}",
-	},
-	"google_iap_tunnel_instance_iam_binding": &importer.SimpleImporter{
-		Fields: []string{"project", "zone", "instance", "role"},
-		Tmpl:   "projects/{{.project}}/iap_tunnel/zones/{{.zone}}/instances/{{.instance}} {{.role}}",
-	},
-	"google_iap_tunnel_instance_iam_member": &importer.SimpleImporter{
-		Fields: []string{"project", "zone", "instance", "role", "member"},
-		Tmpl:   "projects/{{.project}}/iap_tunnel/zones/{{.zone}}/instances/{{.instance}} {{.role}} {{.member}}",
-	},
-	"google_binary_authorization_policy": &importer.SimpleImporter{
-		Fields: []string{"project"},
-		Tmpl:   "projects/{{.project}}",
-	},
-	"google_compute_firewall": &importer.SimpleImporter{
+	"google_storage_bucket": &importer.SimpleImporter{
 		Fields: []string{"project", "name"},
-		Tmpl:   "projects/{{.project}}/global/firewalls/{{.name}}",
+		Tmpl:   "{{.project}}/{{.name}}",
 	},
-	"google_compute_global_address": &importer.SimpleImporter{
-		Fields: []string{"project", "name"},
-		Tmpl:   "projects/{{.project}}/global/addresses/{{.name}}",
+	"google_storage_bucket_iam_binding": &importer.SimpleImporter{
+		Fields: []string{"bucket", "role"},
+		Tmpl:   "{{.bucket}} {{.role}}",
 	},
-	"google_compute_image": &importer.SimpleImporter{
-		Fields: []string{"project", "name"},
-		Tmpl:   "projects/{{.project}}/global/images/{{.name}}",
+	"google_storage_bucket_iam_member": &importer.SimpleImporter{
+		Fields: []string{"bucket", "role", "member"},
+		Tmpl:   "{{.bucket}} {{.role}} {{.member}}",
 	},
-	"google_compute_instance_from_template": &importer.SimpleImporter{
-		Fields: []string{"project", "zone", "name"},
-		Tmpl:   "projects/{{.project}}/zones/{{.zone}}/instances/{{.name}}",
+	"google_storage_bucket_iam_policy": &importer.SimpleImporter{
+		Fields: []string{"bucket"},
+		Tmpl:   "{{.bucket}}",
 	},
-	"google_compute_shared_vpc_host_project": &importer.SimpleImporter{
-		Fields: []string{"project"},
-		Tmpl:   "{{.project}}",
+
+	// GSuite provider
+	"gsuite_group": &importer.SimpleImporter{
+		Fields: []string{"email"},
+		Tmpl:   "{{.email}}",
 	},
-	"google_compute_shared_vpc_service_project": &importer.SimpleImporter{
-		Fields: []string{"host_project", "service_project"},
-		Tmpl:   "{{.host_project}}/{{.service_project}}",
+	"gsuite_group_member": &importer.SimpleImporter{
+		Fields: []string{"group", "email"},
+		Tmpl:   "{{.group}}:{{.email}}",
 	},
+
+	// Helm provider
+	"helm_release": &importer.SimpleImporter{
+		Fields: []string{"namespace", "name"},
+		Tmpl:   "{{.namespace}}/{{.name}}",
+		Defaults: map[string]interface{}{
+			"namespace": "default",
+		},
+	},
+
+	// Kubernetes provider
```

Comment:
p.s. looks like the k8s provider is getting replaced with https://github.com/hashicorp/terraform-provider-kubernetes-alpha, so you may just want to support that...

Reply:
Where do you see that it's getting replaced?

Reply:
Since it's regarded as experimental, I'll see if I can support both.